typings/index.d.ts

```diff
@@ -15,6 +15,11 @@ declare module "discord-starboards" {
       options: StarboardDefaultCreateOptions;
     }
   
+    export interface ColorOptions {
+      colors: string[];
```

Comment:
Maybe rather `colors: (string|number|number[])[]` (I don't know if it's possible)